pkg/controller/node/nodecontroller.go

```diff
@@ -236,6 +236,7 @@ func NewNodeController(
 	podInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc:    nc.maybeDeleteTerminatingPod,
 		UpdateFunc: func(_, obj interface{}) { nc.maybeDeleteTerminatingPod(obj) },
+		DeleteFunc: nc.maybeDeleteTerminatingPod,
```

Comment:
Oh sorry, I had added a pending comment that I wasn't sure this line was right, but I forgot to hit "review changes." (After un-learning the need to "send changes" now I am having trouble adapting to needing to do it...). I'll delete it.